docs/advanced-features/react-18/switchable-runtime.md

```diff
@@ -8,19 +8,6 @@ Next.js has two **server runtimes** where you can render parts of your applicati
 
 By default, Next.js uses the Node.js runtime. [Middleware](https://nextjs.org/docs/advanced-features/middleware) and [Edge API Routes](https://nextjs.org/docs/api-routes/edge-api-routes) use the Edge runtime.
 
-## Global Runtime Option
-
-To configure the runtime for your whole application, you can set the experimental option `runtime` in your `next.config.js` file:
-
-```js
-// next.config.js
-module.exports = {
-  experimental: {
-    runtime: 'experimental-edge', // 'node.js' (default) | experimental-edge
-  },
-}
-```
-
 You can detect which runtime you're using by looking at the `process.env.NEXT_RUNTIME` Environment Variable during runtime, and examining the `options.nextRuntime` variable during compilation.
```

Comment:
Not part of this PR, but the cross usage of the word "runtime" confuses me. I know what this is saying, but i wonder if we could add a short callout that differentiates between the two meanings of the same word?